docker_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+docker run -d -p 8000:8000 fraunhoferiosb/chillimport:$TAG

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=5151228)